hoot-core/src/main/cpp/hoot/core/cmd/DeriveChangesetCmd.cpp

```diff
@@ -98,11 +106,34 @@ class DeriveChangesetCmd : public BaseCommand
       "Deriving changeset for inputs " << input1 << ", " << input2 << " and writing output to " <<
       output << "...");
 
-    //use the same unknown1 status for both so that difference doesn't influence the comparison
+    const double changesetBuffer = ConfigOptions().getChangesetBuffer();
+    if (changesetBuffer > 0.0)
+    {
+      //allow for calculating the changeset with a slightly larger AOI than the default specified
+      //bounding box; useful in certain situations
+      Envelope convertBounds =
+        GeometryUtils::envelopeFromConfigString(ConfigOptions().getConvertBoundingBox());
+      convertBounds.expandBy(changesetBuffer, changesetBuffer);
+      conf().set(
+        ConfigOptions::getConvertBoundingBoxKey(),
+        GeometryUtils::envelopeToConfigString(convertBounds));
+    }
+
+    //some in these datasets may have status=3 if you're loading conflated data, so use
+    //reader.use.file.status and reader.keep.file.status if you want to retain that value
     OsmMapPtr map1(new OsmMap());
     loadMap(map1, input1, true, Status::Unknown1);
     OsmMapPtr map2(new OsmMap());
-    loadMap(map2, input2, true, Status::Unknown1);
+    loadMap(map2, input2, true, Status::Unknown2);
+
+    //we don't want to include review relations
+    boost::shared_ptr<TagKeyCriterion> elementCriterion(
+      new TagKeyCriterion(MetadataTags::HootReviewNeeds()));
+    RemoveElementsVisitor removeElementsVisitor(elementCriterion);
+    removeElementsVisitor.setRecursive(false);
+    map1->visitRw(removeElementsVisitor);
+    map2->visitRw(removeElementsVisitor);
+
```

Comment:
Yeah, you can still do that.  You wouldn't call derive-changeset until the very end of the workflow.  If you call it any earlier than that then you run a higher risk of conflicts with MapEdit data.

Isn't this how we're doing it?:

1) load ref and secondary datasets into hoot;
2) optionally make as many manual changes as you want
3) conflate the data
4) optionally make as many changes as you want
5) resolve all your reviews
6) optionally make as many changes as you want
7) derive your changeset; all reviews are dropped since MapEdit has no concept of a hoot review
8) apply the changeset to MapEdit